### PR TITLE
Accecibilty features

### PR DESCRIPTION
**Pull request by NabilHY**
### Me and my coding partner made changes to my portfolio website so that it's more accessible
These are the points that we checked 

- Page Title: [Updated Page Title](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R24)
- Image text alternatives: [Updated social media icon image `alt`](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R170) and [Work image `alt`](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R62) and [Work image `alt`](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R89) and [Work image `alt`](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R119) and [Work image `alt`](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R143) 
- Text Heading [Updated About me Headers](https://github.com/NabilHY/portfolio1/pull/10/commits/7c90bd4bc569a83abd3be491c1e3ba3c60ae4bca#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R170)
- Color Contrast [Updated Logo contrast](https://github.com/NabilHY/portfolio1/pull/10/commits/0ec192e30991c46056816ca76c713c19573cf0c7#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R44) and [Work section `li` tags](https://github.com/NabilHY/portfolio1/pull/10/commits/0ec192e30991c46056816ca76c713c19573cf0c7#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R157) and [Work section `button` contrast](https://github.com/NabilHY/portfolio1/pull/10/commits/0ec192e30991c46056816ca76c713c19573cf0c7#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R173)
- Resize - no issues found
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <link rel="stylesheet" type="text/css" href="styles1.css" media="screen and (min-width:992px)" />
     <link rel="stylesheet" type="text/css" href="styles2.css" media="screen and (min-width:400px) and (max-width:991px)" />
     <script src="https://kit.fontawesome.com/eb9ab0d66a.js" crossorigin="anonymous"></script>
-    <title>My Portfolio</title>
+    <title>Portfolio's Main Page</title>
```
Comment (NabilHY) [inline index.html:24]: #10 Issue  Found : Previous webpage title was not

File: index.html [hunk 2]
```diff
@@ -167,7 +167,7 @@ <h2>Multi-Post Stories</h2>
 
     <section class="about">
         <div class="deskAbout">
-            <h1>About<br>Myself</h1>
+            <h2>About<br>Myself</h2>
             <p>Hello I'm a software developer! I can help you build a product , feature or website Look
```
Comment (Deepakdanger) [inline index.html:171]: - [ ] From line no. 177 to 188 - All the image `alt` text is similar. Please give alt with appropriate alternative text for all the image.
- [ ] From line 235 to 241 - All input fields are missing form label. Please add label to each input field to resolve this accessibility issue.
- [ ] On line 44, 175, and 226 all the text has contrast issue. Please fix all these three text to make your website more accessible for all kind of users. 

You can check all these issue in the [aditus tool](https://www.aditus.io/button-contrast-checker/) and [WAVE tool extension](https://wave.webaim.org/).

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Benmuiruri & @nabster13 , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Hi @Deepakdanger thanks for the review. :pray: 

@NabilHY and I made the following changes 

- made each image alt unique
- Added the label element for all inputs
- Updated the footer background to improve the contrast.

**Review by elmar8287 — APPROVED** [review]
Hi, @NabilHY.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**